src/main/java/org/lantern/Launcher.java

```diff
@@ -279,26 +279,11 @@ void launch() {
             }
         }
         
-        proxyTracker = instance(ProxyTracker.class);
+        //proxyTracker = instance(ProxyTracker.class);
         httpClientFactory = instance(HttpClientFactory.class);
 
         s3ConfigFetcher = instance(S3ConfigFetcher.class);
         
-        if (checkFallbacks) {
```

Comment:
Okay, thanks. With your changes, it looks like --check-fallbacks simply prevents other services from starting now, similar to --disable-ui:
```java
                // don't need to start the rest of these services when running in check-fallbacks mode
                if (checkFallbacks) {
                    return;
                }
```
I didn't see that command line option being used anywhere else.